WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java

```diff
@@ -280,7 +280,10 @@ public enum UndeletablePrefKey implements PrefKey {
         IS_FIRST_TRY_LOGIN_JETPACK,
 
         // Indicates if this is the first time we try to get the user flags in Jetpack automatically
-        IS_FIRST_TRY_USER_FLAGS_JETPACK
+        IS_FIRST_TRY_USER_FLAGS_JETPACK,
+
+        // Indicates if this is the first time we try to get the user flags in Jetpack automatically
```

Comment:
Would it make sense to replace this part of the comment: `"...to get the user flags..."` with `"...to get the reader saved posts..."` or something like that?

Reply:
copy/paste at play 😄